src/js/components/Carousel/carousel.stories.js

```diff
@@ -22,7 +38,28 @@ const SimpleCarousel = ({ initialChild }) => (
   </Grommet>
 );
 
-storiesOf('Carousel', module).add('Simple', () => <SimpleCarousel />);
-storiesOf('Carousel', module).add('Initial child`', () => (
+const CustomCarousel = () => (
+  <Grommet theme={customTheme}>
+    <Box align="center" pad="large">
+        <Carousel controls='arrows'>
+          <Box pad="xlarge" background="accent-1">
+            <Attraction size="xlarge" />
+          </Box>
+          <Box pad="xlarge" background="accent-2">
+            <TreeOption size="xlarge" />
+          </Box>
+          <Box pad="xlarge" background="accent-3">
+            <Car size="xlarge" />
+          </Box>
+        </Carousel>
+    </Box>
+  </Grommet>
+);
+
+storiesOf('Carousel', module)
+  .add('Simple Carousel', () => <SimpleCarousel />)
+  .add('Without Controls', () => <SimpleCarousel controls={false} play={1000} />)
+  .add('Custom Animation and Styles', () => <CustomCarousel />)
+  .add('Initial child`', () => (
```

Comment:
```suggestion
  .add('Initial Child', () => (
```
small clean up since we are already here :)